burst-kotlin-plugin/src/main/kotlin/app/cash/burst/kotlin/FunctionSpecializer.kt

```diff
@@ -112,7 +113,7 @@ internal class FunctionSpecializer(
       }
     }
 
-    result.annotations += burstApis.testClassSymbol.asAnnotation()
+    result.annotations += testAnnotationClassSymbol.asAnnotation()
```

Comment:
Previously, we added whichever `@Test` we found first Now we add the exact same `@Test` that we stripped off the other function.